Fix terms

hanchao opened the pull request:
The term in the translation file is a comma separated string, not an array of strings

https://github.com/openstreetmap/id-tagging-schema/blob/main/dist/translations/en.json#L3363
```
amenity/school": {
    "name":"School Grounds",
    "terms":"academy,elementary school,middle school,high school"
}
```
![image](https://user-images.githubusercontent.com/1942091/111861119-d78e9200-8986-11eb-8a84-d276f1bacd45.png)

bryceco commented:
This looks great! Because I'm a Swift newbie can you tell me why compactMap is necessary?

hanchao commented:
> This looks great! Because I'm a Swift newbie can you tell me why compactMap is necessary?

terms.split(separator: ",") return [Substring] not [String], so I use compactMap to convert  [String]
```
let substringterms: [Substring] = terms.split(separator: ",")
self.terms = substringterms.compactMap { "\($0)" }
```

bryceco commented:
Makes sense, thanks!
